src/code42cli/main.py

```diff
@@ -50,9 +50,14 @@ def exit_on_interrupt(signal, frame):
 }
 
 
-@click.group(cls=ExceptionHandlingGroup, context_settings=CONTEXT_SETTINGS, help=BANNER)
+@click.group(
+    cls=CLIGroup, context_settings=CONTEXT_SETTINGS, help=BANNER,
+)
+@click.option(
```

Comment:
This option doesn't actually do anything except put it in the help text, since the parsing is short-circuited by the InterpreterGroup.